client/src/modules/list/components/Items/model/reducer.js

```diff
@@ -89,21 +73,21 @@ const items = (state = {}, action) => {
     }
     case ItemActionTypes.UPDATE_SUCCESS: {
       const {
-        payload: {
-          data: { description, name },
-          editedBy,
-          itemId
-        }
+        payload: { description, isOrdered, name, editedBy, _id: itemId }
       } = action;
       const previousItem = state[itemId];
-      const previousDescription = previousItem.description;
-      const newDescription = name ? previousDescription : description;
+      let previousDescription;
+
+      if (previousItem) {
+        previousDescription = previousItem.description;
+      }
 
       return {
         ...state,
         [itemId]: {
-          ...previousItem,
-          description: newDescription,
+          ...(previousItem || action.payload),
+          description: description || previousDescription,
```

Comment:
Why the previous description is kept?

Reply:
Because when you update the just the name and description will come as an empty string or undefined from backend it will actually overwrite the current description.

That's why we have to check if the previous value exists.

Reply:
That behavior is a little bit confusing - what if I wanted to delete description from 'Test' to an empty text ''?

Reply:
Could we create some helper/helpers to filter ous undefined props or 'empty' strings and the use the filtred object to avoid those multiple checks?

Reply:
we have already helper for that ->` isDefined()`

Reply:
> That behavior is a little bit confusing - what if I wanted to delete description from 'Test' to an empty text ''?

You right, now it works correctly :)